### PR TITLE
[US30 Task85] Code to get Closed User Stories with Zero Business Value

### PR DESCRIPTION
**Pull request by AdvaitChirmule**

### CODE REVIEW

File: taigaProject/src/taigaApi/util/metric_service.py [hunk 1]
```diff
@@ -210,3 +210,24 @@ def append_points_date_data(date, completed_story_points, remaining_story_points
         "completed": completed_story_points,
         "remaining": remaining_story_points
     }
+
+
+def get_zero_business_value_user_stories(project_details, start_range, end_range, attribute_key, auth_token):
```
Comment (shahih25) [inline taigaProject/src/taigaApi/util/metric_service.py]: Can you please create an API in the metric controller and then call this method from there?

Comment (AdvaitChirmule) [reply taigaProject/src/taigaApi/util/metric_service.py]: I have made the changes

File: taigaProject/src/taigaApi/controller/metric_controller.py [hunk 2]
```diff
@@ -33,3 +34,8 @@ def get_cycle_time_metric(projectRequest: ProjectRequest, token: Annotated[str |
 @router.post("/metric/Burndown")
 def get_burndown_chart_metric(burndownChartRequest: BurndownChartRequest, token: Annotated[str | None, Header()] = None):
     return get_burndown_chart_metric_detail(burndownChartRequest.milestoneId, burndownChartRequest.attributeKey, token)
+
+@router.post("/metric/Cruft")
+def get_zero_business_value(cruftRequest: CruftRequest, token: Annotated[str | None, Header()] = None):
+    project_info = get_project_by_slug(cruftRequest.projectSlug, token)
```
Comment (shahih25) [inline taigaProject/src/taigaApi/controller/metric_controller.py]: You can get the project id from UI along with the start and end dates. So you can remove this call from here.
Also, change **projectSlug** to **project_id** in the CruftRequest.

File: taigaProject/src/taigaApi/model/cruft.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+class CruftRequest(BaseModel):
+    projectSlug: str = None
```
Comment (shahih25) [inline taigaProject/src/taigaApi/model/cruft.py]: change it to **project_id**

File: taigaProject/src/taigaApi/controller/metric_controller.py (hunk 2 above)
Comment (AdvaitChirmule) [reply taigaProject/src/taigaApi/controller/metric_controller.py]: Made the change

File: taigaProject/src/taigaApi/model/cruft.py (hunk 3 above)
Comment (AdvaitChirmule) [reply taigaProject/src/taigaApi/model/cruft.py]: Renamed to projectId

File: taigaProject/src/taigaApi/util/metric_service.py (hunk 1 above)
Comment (shahih25) [inline taigaProject/src/taigaApi/util/metric_service.py]: This will also change.

Comment (AdvaitChirmule) [reply taigaProject/src/taigaApi/util/metric_service.py]: Right, fixed

**Review by shahih25 — APPROVED** [review]
lgtm

**Comment by spark290** [comment]
LGTM

**Comment by spark290** [comment]
LGTM

**Review by spark290 — APPROVED** [review]
LGTM

**Pull request merged**